lib/services/files/storage/index.js

```diff
@@ -5,17 +5,31 @@ import * as local from './local';
 
 let fsRepo;
 
+export const RACKSPACE = 'rackspace';
+export const AMAZON = 'amazon';
+export const GOOGLE = 'google';
+export const LOCAL = 'local';
+
 switch (process.env.FS_REPO) {
-  case 'rackspace': fsRepo = rackspace; break;
-  case 'amazon': fsRepo = amazon; break;
-  case 'google': fsRepo = google; break;
+  case RACKSPACE: fsRepo = rackspace; break;
+  case AMAZON: fsRepo = amazon; break;
+  case GOOGLE: fsRepo = google; break;
   default: fsRepo = local; break;
 }
 
 const uploadFromStream = fsRepo.uploadFromStream;
 const uploadFromPath = fsRepo.uploadFromPath;
 const downloadToStream = fsRepo.downloadToStream;
 
```

Comment:
Lets add the createDeleteFromPath to here and export it directly out the class, for anything that needs it implicitly